pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -107,6 +107,11 @@ func (l *PodInterface) PlugPhase1(vmi *v1.VirtualMachineInstance, iface *v1.Inte
 			return err
 		}
 
+		isMultiqueue := (vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue != nil) && (*vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue)
+		if err = driver.createTapDevice(isMultiqueue); err != nil {
```

Comment:
What is the reason behind moving `createTapDevice` outside of `preparePodNetworkInterfaces`. All the networking preparation is done in `preparePodNetworkInterfaces`. Why creation of the bridge should be in `preparePodNetworkInterfaces` and creation of the tap outside of it?

Reply:
I think you're right; I just didn't want to polute the `preparePodNetworkInterfaces` signature with the `isMultiqueue` parameter - as such, moved that out.

I'll take another look into it, but what you suggest might be the least worse option.